guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The initialized model <s>would</s> <u>will </u>need a learning rate for training, and the <code>lr_find()</code> method <s>is</s> <u>can be </u>used <s>for searching</s> <u>to search for </u>an optimal learning rate <s>for passing it to</s> <u>to pass into </u>the final model fitting as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>